README.md

```diff
@@ -0,0 +1,149 @@
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Todo List App <a name="about-project"></a>
+
+**Todo List App** is a website for Where you can keep track of your daily Todos.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Build With</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Do all your maths with this web app**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://github.com/gsmalik030/todo-list-react) [Coming soon]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd math-magicians
+  git clone https://github.com/gsmalik030/todo-list-react
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd math-magicians
+  git clone https://github.com/gsmalik030/todo-list-react
+  npm install
+```
+
+### Usage
+
+Use this app by runing:
+
+```sh
+   cd math-magicians
+  git clone https://github.com/gsmalik030/todo-list-react
+  npm install
+  npm start
+```
+
+### Run tests
+
+Test this app by runing:
+
+```sh
+  cd math-magicians
+  git clone https://github.com/gsmalik030/todo-list-react
+  npm install
+  npm test
+```
+
+### Deployment
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Ghulam Subbhani(Malik)**
+
+- GitHub: [@githubhandle](https://github.com/gsmalik030)
+- Twitter: [@twitterhandle](https://twitter.com/gsmalik030)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ghulam-subbhani-4b1281252/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add functionality**
+- [ ] **Add styles**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/gsmalik030/todo-list-react/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please, give a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank to all my Microverse partners.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](MIT.md) licensed.
```

Comment:
Please try to make sure that your MIT link actually links to the MIT license that is already in your project.